packages/app/src/pages/PoolPage/RemoveLiquidity.tsx

```diff
@@ -60,6 +44,34 @@ export default function RemoveLiquidityPage() {
     return null;
   }
 
+  const validateRemoveLiquidity = () => {
+    const errors = [];
+
+    if (!tokenInput.amount) {
+      errors.push(`Enter ${liquidityToken.name} amount`);
+    }
+    if (!tokenInput.hasEnoughBalance) {
+      errors.push(`Insufficient ${liquidityToken.name} balance`);
+    }
+
+    return errors;
+  };
+  const errorsRemoveLiquidity = validateRemoveLiquidity();
```

Comment:
I think this isn't a good practice maybe, run this type of function inside the component scope. You could run this inside the `onInput` method of `useInput()`

Reply:
inside `onInput` doesn't work because we need other values than the input, and we cannot assume they will be right when the function is running.

I can do a `useEffect` , listening only to the props that affects the validation, what you think @pedronauck ??